miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/view/MiniAppParameters.kt

```diff
@@ -6,7 +6,9 @@ import com.rakuten.tech.mobile.miniapp.MiniAppInfo
 /**
  * This class can be used in the HostApp to create the mini app views using mini app id.
  */
-sealed class MiniAppParameters {
+sealed class MiniAppParameters(
+    open val config: MiniAppConfig,
```

Comment:
```suggestion
```
can you change it to these,  adding overridden parameter in the `sealed` class is not necessary now, i am a bit concerned that `config` might be effected the parameters inside it and the miniapps itself, but we can improve this later with some more tests. thank you.

Reply:
Thanks for the comment,  it was covered in the unit tests and it goes well. Would you explain what kind of effect that be?

do you mean to remove it back to the origin and suggest doing the same lines of code of finding which type of `MiniAppParameters` as in the `load()`?

Reply:
yes @shohiebsense